grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancer.java

```diff
@@ -54,8 +53,9 @@
   private GrpclbConfig config = DEFAULT_CONFIG;
 
   // All mutable states in this class are mutated ONLY from Channel Executor
+  @VisibleForTesting
```

Comment:
There is a getter visible for testing already.

Reply:
oops done.